src/eggd_mosdepth.sh

```
@@ -57,38 +62,42 @@ main() {
          export MOSDEPTH_Q2=CALLABLE;
          export MOSDEPTH_Q3=HIGH_COVERAGE;
-         mosdepth $optional_arguments $bam_prefix '/data/input/$bam_prefix.bam'"
+         mosdepth $optional_arguments $fasta $alignment_file_prefix '/data/$input_alignment_file'"
         fi
+
     else
     # not using quantize option, run mosdepth normally with container
-    sudo docker run -v `pwd`:/data -w "/data/out/mosdepth_output" $mosdepth_id mosdepth $optional_arguments $bam_prefix "/data/input/$bam_prefix.bam"
+
+    # set paths to inputs for Docker
+    input_alignment_file=$(find input/$alignment_file.*am)
```

Shellcheck (0.10.0)

[warning] alignment_file is referenced but not assigned.

(SC2154)